### DISCUSSION TITLE
Is it possible to change the rounded border for all the UI elements?

### DISCUSSION THREAD

**Discussion by freddiehaddad** [mrcjkb/rustaceanvim#362] [Q&A] [answered]
I searched through the help and found the option for:

```lua
tools = {
  float_win_config = {
    border = 'double',
  },
}
```

But this doesn't work for code actions, etc.  I can see in the code that it looks hard-coded to `rounded` here:

https://github.com/mrcjkb/rustaceanvim/blob/e5415b59ffa6c00e04c2119561486617379e3766/lua/rustaceanvim/commands/code_action_group.lua#L173

Is there an easy way to override that?

Setting the handler as follows doesn't work:

```lua
vim.lsp.handlers['textDocument/codeAction'] = vim.lsp.with(vim.lsp.buf.code_action, {})
```

**Comment by mrcjkb** [comment, marked as answer]
Hey 👋 

I've opened https://github.com/mrcjkb/rustaceanvim/issues/363
